master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -64,7 +64,9 @@ func (a *agentResourceManager) Receive(ctx *actor.Context) error {
 	case sproto.ResourcesReleased:
 		a.forwardToAllPools(ctx, msg)
 
-	case sproto.SetGroupMaxSlots, sproto.SetGroupWeight, sproto.SetGroupPriority:
+	case sproto.SetGroupMaxSlots, job.SetGroupWeight, job.SetGroupPriority:
+		// QUESTION do we wanna be forwarding these to all pools? they don't seem to
```

Comment:
Good question - is there confirmation this is not a problem?

Reply:
I think it ends up creating groups for those jobs on all resource pools which end up doing nothing. this is existing behavior but we might need to address it when we start switching jobs between resource pools